values.yaml

```diff
@@ -0,0 +1,1742 @@
+## Default values for Datadog Agent
```

Comment:
I'm not sure this file should be in the root folder of repo. Even if we need have example of datadog helm configuration will be better to place it into cloudify-manager-worker/external/datadog.yaml (for example) and add a section inside README.md about it (what is it, how to install datadog using this config, important configuration parameters, etc).